Makefile

```diff
@@ -0,0 +1,20 @@
+OJS := $(wildcard *.js)
+MJS := $(OJS:.js=.min.js)
+DJS := $(addprefix dist/,${MJS})
+
+all: $(DJS)
+.PHONY: all
+
+dist/%.min.js : %.js
+	npx uglify-js $^ --compress --mangle --no-annotations -o $@ --source-map
+
+format:
+	npx prettier -w *.js
+
+install:
+	yarn add -D uglify-js prettier
```

Comment:
is there a reason why you don't want to use a package.json for dependencies?
it would be better to manage versions, no?

Reply:
Since it's just for for cli tools, I figured why bother. But I'll create one if you prefer

Reply:
I'm fine with what you did, don't worry :)